docs/unifiedpush/push-message-format/index.markdown

```diff
@@ -64,6 +78,10 @@ None! The JSON map is submitted as it is, directly to the device. There are no A
 
 To make the user experience the same on iOS and Android, for cordova users, we use the iOS alert 'key' on Android as well to generate a notification for you. And we've introduced a 'title' that can optionally be used for the title of this notification, if none is specified the application name will be used like on iOS.
 
+##### Windows special keys
+
+For windows we also support sending toast, tile, badge and raw notifications for MPNS and WNS to keep the message uniform the first text for the message template is the value from alert and for a numeric badge we use the main badge field. Of course you could add criteria to only select windows devices. If you don't specify the windows section of the message a toast notification is send with the alert as the content. More information about the [windows message format](windows-document-format/)
```

Comment:
Let's add a `.` after `raw notifications for MPNS and WNS` and starta new sentence `To keep ... `